prime-angular-frontend/src/app/modules/auth/pages/phsa/phsa.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Inject } from '@angular/core';
+import { ActivatedRoute, Data } from '@angular/router';
+
+import { APP_CONFIG, AppConfig } from 'app/app-config.module';
+import { ViewportService } from '@core/services/viewport.service';
+import { IdentityProviderEnum } from '@auth/shared/enum/identity-provider.enum';
+import { AuthService } from '@auth/shared/services/auth.service';
+import { PhsaLabtechRoutes } from '@phsa/phsa-labtech.routes';
+
+@Component({
+  selector: 'app-phsa',
+  templateUrl: './phsa.component.html',
+  styleUrls: [
+    './phsa.component.scss',
+    '../../shared/styles/landing-page.scss']
+})
+export class PhsaComponent implements OnInit {
+
+  private routeData: Data;
+
+  constructor(
+    @Inject(APP_CONFIG) private config: AppConfig,
+    private authService: AuthService,
+    private viewportService: ViewportService,
+    activatedRoute: ActivatedRoute
+  ) {
+    this.routeData = activatedRoute.snapshot.data;
+  }
+
+  public get loginButtonMarkup(): string {
+    return this.routeData.button;
+  }
+
+  public get subtitle(): string {
```

Comment:
You can definitely provide this through a getter, but there's actually a bit of performance to be had if it can be made into a member variable since this method will be invoked on every digest cycle.  If it was me I'd probably have this as a `public title: string;` and then set in the constructor.